pages/index.jsx

```diff
@@ -1,14 +1,63 @@
 import React from 'react';
 import Layout from '../components/Layout';
-import NavBar from '../components/Navbar';
-import Map from '../components/Map';
+import {
+  Col,
+  Row,
+  Navbar,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  TabContent,
+} from 'reactstrap';
+import MapFilters from '../components/MapFilters';
 
 export default class IndexPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      activeTab: '1',
+    };
+  }
+  toggle(tab) {
```

Comment:
`toggle` isn’t as descriptive as it could be. `switchTab`?

Reply:
agreed - updated name